thunder/tests/test_grad.py

```diff
@@ -327,7 +328,8 @@ def check_vjp(f, *primals, executor="torch", atol=1e-5, rtol=1.3e-6):
     if J_u_v.isnan().any():
         # TODO: find a better way to handle NaNs in finite differences
         return  # skip this sample
-    torch.testing.assert_close(J_u_v, u_J_star_v, atol=atol, rtol=rtol, check_device=False)
+    partial_comp = partial(comp, atol=atol, rtol=rtol, check_device=False)
```

Comment:
I do wonder if atol and rtol and check_device should be applied to custom comparison functions, or whether, in fact, the custom comparison function could replace passing non-default atol/rtol (because that's what we are using it for after all).

Reply:
Observing the most use-cases for custom comparators, it seems like we should have atol, rtol, and check_device. If not, I think opinfos.py should be modified accordingly. Are you expecting custom comparators to be used in a different way?

Reply:
I would expect the custom comparator to do these things with it's own values and not feed atol/rtol to it (and I'm not sure whether the current PR would override the atol/rtol in the custom comparator's signature).
I was hoping to use the `test_ops` for comparison, I think there is this TODO: 
https://github.com/Lightning-AI/lightning-thunder/blob/21a222b180009616a4cc48176958b4506894a330/thunder/tests/test_ops.py#L52C1-L52C2

Reply:
I see what you mean! Thanks for the explanation. I think what you said (`todo these things with it's own values and not feed atol/rtol to it`) makes more sense. Let me update this!